src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/DataFlowOperationVisitor.cs

```diff
@@ -3638,6 +3639,11 @@ public sealed override TAbstractAnalysisValue VisitLocalFunction(ILocalFunctionO
         /// </summary>
         protected INamedTypeSymbol? GenericIEquatableNamedType { get; }
 
+        /// <summary>
+        /// <see cref="INamedTypeSymbol"/> for <see cref="System.IO.StringReader"/>
+        /// </summary>
+        protected INamedTypeSymbol? StringReaderType { get; }
```

Comment:
You need to add this to the public API surface for AnalyzerUtilities package.

Reply:
`F:\workspace\_work\1\s\src\Utilities\FlowAnalysis\FlowAnalysis\Framework\DataFlow\DataFlowOperationVisitor.cs(3645,56): error RS0016: Symbol 'StringReaderType.get' is not part of the declared API.`

Reply:
Yes sorry I missed the warning in VS :disappointed: